pkg/limayaml/limayaml.go

```diff
@@ -43,3 +45,8 @@ type Video struct {
 	// Display is a QEMU display string
 	Display string `yaml:"display,omitempty"`
 }
+
+type Provision struct {
+	System string `yaml:"system,omitempty"`
+	User   string `yaml:"user,omitempty"`
+}
```

Comment:
Would it make sense to support multiple scripts?

```go
type LimaYAML struct {
	Provision []Provison `yaml:"provision,omitempty"`
}

type ProvisionMode = string
const (
  ProvisionModeSystem ProvisonMode = "system"
  ProvisionModeUser ProvisonMode = "user"
)
type Provision struct {
	Mode ProvisionMode `yaml:"mode"` // default: "system"
        Script string `yaml:"script"`
}
```

Reply:
"readiness" script should have similar structure.

Reply:
I don't know; since they are user-provided, I thought the user can always combine them into a single script.

What I can potentially see though is that somebody wants to distribute a `lima.yaml` for a special VM setup (with custom provisioning scripts), and the final end-user wants to further customize it. Then it might be cleaner for the end-user to be able to provide a separate script.

I can implement it, if you prefer it.